services/data-info/src/data_info/routes/data.clj

```diff
@@ -130,4 +132,13 @@ with characters in a runtime-configurable parameter. Currently, this parameter l
     "ERR_INVALID_JSON, ERR_EXISTS, ERR_DOES_NOT_EXIST, ERR_NOT_READABLE,"
     "ERR_NOT_WRITEABLE, ERR_NOT_A_USER, ERR_BAD_PATH_LENGTH, ERR_BAD_DIRNAME_LENGTH,"
     "ERR_BAD_BASENAME_LENGTH, ERR_TOO_MANY_RESULTS"))
-        (svc/trap uri meta/do-metadata-save data-id params body)))))
+        (svc/trap uri meta/do-metadata-save data-id params body))
+
+      (GET* "/permissions" [:as {uri :uri}]
```

Comment:
If you'd rather leave this to me migrating sharing more concretely later that's fine, but I've been trying to move more stuff out of this file (which sort of became a dumping ground for routes). I'd prefer to have this in the same file as `/permissions-gatherer`, which is currently `routes.users` but in line with your other `users` -> `permissions` changes could be moved quite reasonably.